front_end/src/components/history/history.css

```diff
@@ -218,6 +218,13 @@
   text-align: center;
```

Comment:
This css classes wasn't used anywhere: HistoryPageText2

Reply:
removed